.github/contribution/testing-conventions.md

```diff
@@ -87,6 +87,39 @@ Commands:
 
 - `yarn components test-ct` - test components, both behaviour and screenshots
 - `yarn components test-ct --update-snapshots` - update screenshots
+- `yarn components test-ct path/to/a/component` - test a single component
+
+### Writing the tests
+
+The tests are written in `*.ct.tsx` files next to the component. The test file should have the
+following structure:
+
+```tsx title="src/components/Component/Component.ct.tsx"
```

Comment:
is "tsx title=".." changes something? I didn't notice when opened it, maybe having a comment below is also a good idea, as we had it in Next docs.

```suggestion
```tsx title="src/components/Component/Component.ct.tsx"
// src/components/Component/Component.ct.tsx
```

Reply:
Just copilot things left unnoticed. Will add the comment :+1:

Reply:
While on this, I noticed that I could add an example to the story file as well. So will do, be sure to review it as well 🙏